packages/manager/modules/cloud-universe-components/src/price/service.js

```diff
@@ -44,7 +45,12 @@ export default class OvhCloudPriceHelper {
         forEach(projectPlan.addonFamilies, (family) => {
           forEach(family.addons, (planCode) => {
             pricesMap[planCode] = this.transformPrice(
-              get(find(catalog.addons, { planCode }), 'pricings[0]'),
+              find(
+                get(find(catalog.addons, { planCode }), 'pricings'),
+                ({ capacities }) =>
+                  includes(capacities, 'renew') ||
```

Comment:
```
capacities.includes('renew') || capacities.includes('consumption')
```
? To avoid an import ^^

Reply:
As capacities is in the structure of the object returned by API, we'll need to check it's defined first

Reply:
@Jisay yes but it implies checking if capacities is an array and other things :)